lib/site_template/404.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+---
+
+<style type="text/css" media="screen">
+  .container {
+    margin: 10px auto;
+    max-width: 600px;
+    text-align: center;
+  }
+  h1 {
+    margin: 30px 0;
+    font-size: 4em;
+    line-height: 1;
+    letter-spacing: -1px;
+  }
+</style>
+
+<div class="container">
+  <h1>404</h1>
+
+  <p><strong>Page not found :(</strong></p>
+  <p>The requested page could not be found.</p>
+
+  <p>Edit the <code>404.html</code> file in your site to show a custom 404 Error Message.</p>
```

Comment:
I intended it to be similar to the imperative voice in default `{{ site.description }}`